x/btcstaking/types/incentive.go

```diff
@@ -56,6 +56,9 @@ func (dc *VotingPowerDistCache) ApplyActiveFinalityProviders(maxActiveFPs uint32
 		if !fp.IsTimestamped {
 			break
 		}
+		if fp.IsJailed {
+			break
+		}
```

Comment:
Given all these new jailed/timestamped concepts, I started to feel `TotalVotingPower` is no longer an accurate name. How about we rename it to sth like `TotalBTCStake`? Not for this PR, just some thought

Reply:
Good point. Tracked in https://github.com/babylonlabs-io/babylon/issues/72. Will fix it in a separate PR